net/nettest/conn.go

```diff
@@ -5,21 +5,13 @@
 package nettest
 
 import (
-	"io"
+	"net"
 	"time"
 )
 
 // Conn is a bi-directional in-memory stream that looks like a TCP net.Conn.
```

Comment:
this isn't really how you document an interface, but this was here before.

How about:

// Conn is a net.Conn that can additionally have its reads and writes blocked and unblocked.